source/macos/AdaptiveCards/AdaptiveCardsTests/Fakes/FakeRichTextBlock.swift

```diff
@@ -27,9 +27,13 @@ class FakeRichTextBlock: ACSRichTextBlock {
 }
 
 extension FakeRichTextBlock {
-    static func make(textRun: FakeTextRun = FakeTextRun.make(), horizontalAlignment: ACSHorizontalAlignment = ACSHorizontalAlignment.left, heightType: ACSHeightType = .auto) -> FakeRichTextBlock {
+    static func make(textRun: FakeTextRun = FakeTextRun.make(), horizontalAlignment: ACSHorizontalAlignment = ACSHorizontalAlignment.left, heightType: ACSHeightType = .auto, inlines: [ACSInline]? = nil) -> FakeRichTextBlock {
         let fakeRichTextBlock = FakeRichTextBlock()
-        fakeRichTextBlock.inlines = [textRun as ACSInline]
+        if let inlines = inlines {
```

Comment:
```suggestion
        if let inlines = inlines, !inlines.isEmpty {
```

Reply:
should need this?

Reply:
No I intentinally added this to check how renderer behaves when we pass empty inlines, if we do this then we will not be testing the renderer itself

Reply:
Ok, hope it will make sense. LGTM.